cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/AnomalyDetector.java

```diff
@@ -295,6 +296,22 @@ private boolean isReadyToFix(Anomaly anomaly) {
       return false;
     }
 
+    private void mayBePersistSelfHealingInfo(Anomaly anomaly, boolean selfHealingStarted) {
```

Comment:
this method in `AnomalyDetector` only log successfully started task, but `UserTaskManager`  logs both successfully started and fail to start task(task completed exceptionally), is this intended?

Reply:
Will add log for failure case